content/pt-br/docs/setup/production-environment/tools/kubespray.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Instalando Kubernetes com Kubespray
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+Este início rápido ajuda a instalar um cluster Kubernetes hospedado no GCE, Azure, OpenStack, AWS, vSphere, Equinix Metal (anteriormente Packet), Oracle Cloud Infrastructure (Experimental) ou Baremetal com [Kubespray](https://github.com/kubernetes-sigs/kubespray).
```

Comment:
Suggestion: use "em servidor dedicado" instead of baremetal (see https://pt.wikipedia.org/wiki/Bare-metal_server).